### ISSUE TITLE
php segfaults when mocking class with __call

### ISSUE THREAD

**Issue by zodiac** [mockery/mockery#564]
php 5.5.33 segmentation faults when running the following code

```
<?php

require('/path/to/vendor/autoload.php');

class CommandRunner {
    public function __call($name, $args) {
        return 1;
    }
}

$cr = Mockery::mock('CommandRunner')->makePartial();
$cr->fopen("/tmp/nonexistent", 'r');
```

I know mockery doesn't support passing through to non-real methods (http://docs.mockery.io/en/latest/reference/mockery/gotchas.html) but a segfault seems excessively harsh

**Comment by davedevelopment** [comment]
Confirmed, not been able to run with a debugger yet.

**Comment by robertbasic** [comment]
The PR got merged, so this issue can be closed @davedevelopment!

**Comment by robertbasic** [comment]
Closing as #584 was merged.

**Comment by alexweissman** [comment]
It would appear that this has not been completely resolved - see #777.  I am using the latest release of Mockery (0.9.9).

**Comment by alexweissman** [comment]
Nevermind, just realized that 1.0.0-alpha existed!  Seems to have fixed the issue.
